setup/users/create_users.go

```diff
@@ -69,7 +69,7 @@ func getMemberClusterName(cl client.Client, hostOperatorNamespace, memberOperato
 	return memberCluster.Name, err
 }
 
-func containsClusterCondition(conditions []toolchainv1alpha1.ToolchainClusterCondition, contains *toolchainv1alpha1.ToolchainClusterCondition) bool {
+func containsClusterCondition(conditions []toolchainv1alpha1.Condition, contains *toolchainv1alpha1.Condition) bool {
```

Comment:
We don't need this special function anymore, do we? We can use the other generic function used for the standard Toolchain condition

Reply:
I didn't understand it completely. Does this mean to remove this function? i dont see a generic function though which can be used in its place..? and hence i kept this

Reply:
yes, you can remove this function and change the place where it is being called to:
```go
if condition.IsTrue(cluster.Status.Conditions, toolchainv1alpha1.ToolchainClusterReady) {
	memberCluster = cluster
	return true, nil
}
```
`condition.IsTrue` is the generic function I was referring to

Reply:
ohk got it, i somehow thought there is generic function for standard toolchain condition, which i couldn't find and hence the confusion , thanks for clarifying